plugins/event_source/k8s.py

```diff
@@ -604,9 +604,10 @@ def get_nested_value(d, keys):
                                         "{}",
                                     )
                                 )
-                                last_applied_configuration = json.loads(
-                                    last_applied_configuration_str
-                                )
+                                if last_applied_configuration_str:
+                                    last_applied_configuration = json.loads(
+                                        last_applied_configuration_str
+                                    )
```

Comment:
May want to add a debug log here. After that, continue to avoid falling into the other logic.